### PR TITLE
Added new worldmap widget setting:

### PR DESCRIPTION
**Pull request by remyj38**
Signed-off-by: Rémy Jacquin <remy@remyj.fr>

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

----
Added new worldmap widget setting:
The user can chose how down devices under maintenance are shown: as a down device (default and as currently), as a warning device (orange icon) or as a up device. Up devices and warning devices are not shown if the map is configured to show only down devices

### CODE REVIEW

**Comment by kkrumm1** [comment]
@remyj38 can you add a little tidbit about this to the world map doc, please. 
https://github.com/librenms/librenms//edit/master/doc/Extensions/World-Map.md

**Comment by remyj38** [comment]
@kkrumm1 Done 😄

File: html/includes/common/worldmap.inc.php [hunk 1]
```diff
@@ -93,6 +95,30 @@
             </select>
         </div>
     </div>
+    <div class="form-group">
+        <div class="col-sm-4">
+            <label for="maintenance" class="control-label">Down devices over maintenance should be shown as: </label>
```
Comment (laf) [inline html/includes/common/worldmap.inc.php]: Maybe `Down devices in maintenance windows shown as:`

**Comment by murrant** [comment]
Actually, why not show devices under maintenance as a totally different color?  We wouldn't need an option then.

Perhaps light blue.  Would also need a tooltip to note the status.

**Comment by kkrumm1** [comment]
I agree with murrant because in the Availability map Device is shown as Yellow as warning for restarted in last 24 hours. This could be confusing with the world map having a color of orange. Users may think that is because of the warning (restarted).

File: html/includes/common/worldmap.inc.php (hunk 1 above)
Comment (remyj38) [reply html/includes/common/worldmap.inc.php]: @laf: I removed option in 288edca

**Comment by remyj38** [comment]
@murrant @kkrumm1 : I removed the option and changed the color to light blue.

**Comment by laf** [comment]
Thanks very much for this @remyj38.

**Pull request merged**